src/MgvView.sol

```diff
@@ -108,6 +108,13 @@ contract MgvView is MgvCommon {
 
   // # Offer view functions
 
+  /* Get the olKey that corresponds to a hash, only works for offerLists that have been activated > 0 times */
+  function olKeys(bytes32 olKeyHash) external view returns (OLKey memory olKey) {
+    unchecked {
+      olKey = _olKeys[olKeyHash];
+    }
+  }
+
```

Comment:
I'd say this is an offer LIST view function, so should be listed with those ;-)

Reply:
fixed! 141636413fb7a17b04097c3cde56559e5c124dde